client/src/js/dd_main.js.map

```diff
@@ -1 +1 @@
AAC,CAAC;IACrB,WAAW,CAAC,KAAK,CAAC,GAAG,CAAC,CAAC;IACvB,YAAY,CAAC,KAAK,CAAC,CAAC,CAAC,CAAC;IACtB,YAAY,CAAC,KAAK,CAAC,CAAC,CAAC,CAAC;IACtB,IAAM,IAAI,GAAG,IAAI,IAAI,EAAE,CAAC;IACxB,QAAQ,CAAC,cAAc,CAAC,QAAQ,CAAC,CAAC,SAAS;QACvC,IAAI,CAAC,kBAAkB,CAAC,EAAE,EAAE,EAAC,IAAI,EAAE,SAAS,EAAE,MAAM,EAAE,SAAS,EAAC,CAAC,CAAC;IAItE,MAAM,CAAC,WAAW,CAAC;QACjB,IAAM,SAAS,GAAG,IAAI,IAAI,EAAE,CAAC;QAC7B,QAAQ,CAAC,cAAc,CAAC,QAAQ,CAAC,CAAC,SAAS;YACvC,SAAS,CAAC,kBAAkB,CAAC,EAAE,EAAE,EAAC,IAAI,EAAE,SAAS,EAAE,MAAM,EAAE,SAAS,EAAC,CAAC,CAAC;IAC7E,CAAC,EAAE,KAAK,CAAC,CAAC;IAEV,MAAM,CAAC,gBAAgB,CAAC,QAAQ,EAAE;QAChC,KAAK,CAAC,MAAM,EAAE,CAAC;QACf,YAAY,CAAC,MAAM,EAAE,CAAC;QACtB,YAAY,CAAC,MAAM,EAAE,CAAC;QACtB,SAAS,CAAC,MAAM,EAAE,CAAC;QACnB,WAAW,CAAC,MAAM,EAAE,CAAC;IACvB,CAAC,CAAC,CAAC;IAEH,IAAM,WAAW,GAAG,CAAC,CAAC;IACtB,IAAM,YAAY,GAAG,CAAC,CAAC;IACvB,IAAM,UAAU,GAAG,CAAC,CAAC;IACrB,IAAM,WAAW,GAAG,CAAC,CAAC;IACtB,IAAM,QAAQ,GAAG,CAAC,CAAC;IACnB,IAAM,SAAS,GAAG,CAAC,CAAC;IACpB,IAAM,eAAe,GAAG,CAAC,CAAC;IAC1B,IAAM,eAAe,GAAG,CAAC,CAAC;IAC1B,IAAM,YAAY,GAAG,CAAC,CAAC;IACvB,IAAM,QAAQ,GAAG,CAAC,CAAC;IACnB,IAAM,WAAW,GAAG,EAAE,CAAC;IACvB,IAAM,SAAS,GAAG,EAAE,CAAC;IACrB,IAAM,KAAK,GAAG,EAAE,CAAC;IAEjB,IAAM,EAAE,GAAG,IAAI,SAAS,CAAC,wBAAwB,CAAC,CAAC;IACnD,EAAE,CAAC,SAAS,GAAG,UAAC,KAAK;QACnB,IAAM,GAAG,GAAG,IAAI,CAAC,KAAK,CAAC,KAAK,CAAC,IAAI,CAAC,CAAC;QACnC,OAAO,CAAC,GAAG,CAAC,GAAG,CAAC,CAAC;QAEjB,MAAM,CAAC,CAAC,GAAG,CAAC,EAAE,CAAC,CAAC,CAAC;YACf,KAAK,eAAe;gBAClB,YAAY,CAAC,KAAK,CAAC,GAAG,CAAC,IAAI,CAAC,CAAC;gBAC7B,KAAK,CAAC;YACR,KAAK,eAAe;gBAClB,WAAW,CAAC,OAAO,CAAC,EAAC,CAAC,EAAE,GAAG,CAAC,SAAS,EAAE,CAAC,EAAE,GAAG,CAAC,IAAI,EAAC,CAAC,CAAC;gBACrD,YAAY,CAAC,KAAK,CAAC,GAAG,CAAC,IAAI,CAAC,CAAC;gBAC7B,KAAK,CAAC;YACR,KAAK,KAAK;gBACR,SAAS,CAAC,KAAK,CAAC,GAAG,CAAC,IAAI,CAAC,CAAC;gBAC1B,KAAK,CAAC;YACR,KAAK,YAAY;gBACf,WAAW,CAAC,KAAK,CAAC,GAAG,CAAC,IAAI,CAAC,CAAC;gBAC5B,KAAK,CAAC;YACR,KAAK,WAAW;gBACd,KAAK,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,KAAK,CAAC,CAAC;gBAC7B,KAAK,CAAC;YACR,KAAK,YAAY;gBACf,KAAK,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,MAAM,CAAC,CAAC;gBAC9B,KAAK,CAAC;YACR,KAAK,UAAU;gBACb,IAAI,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,KAAK,CAAC,CAAC;gBAC5B,KAAK,CAAC;YACR,KAAK,WAAW;gBACd,IAAI,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,MAAM,CAAC,CAAC;gBAC7B,KAAK,CAAC;YACR,KAAK,QAAQ;gBACX,IAAI,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,KAAK,CAAC,CAAC;gBAC5B,KAAK,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,KAAK,CAAC,CAAC;gBAC7B,KAAK,CAAC;YACR,KAAK,WAAW;gBACd,IAAI,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,MAAM,CAAC,CAAC;gBAC7B,KAAK,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,MAAM,CAAC,CAAC;gBAC9B,KAAK,CAAC;YACR,KAAK,SAAS;gBACZ,MAAM,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,MAAM,CAAC,CAAC;gBAC/B,KAAK,CAAC;YACR,KAAK,QAAQ;gBACX,MAAM,CAAC,KAAK,CAAC,GAAG,CAAC,KAAK,CAAC,KAAK,CAAC,CAAC;gBAC9B,QAAQ,CAAC,cAAc,CAAC,cAAc,CAAC,CAAC,SAAS;oBAC7C,SAAS,CAAC,KAAK,EAAE,CAAC,QAAQ,EAAE,CAAC;gBACjC,KAAK,CAAC;YACR,KAAK,WAAW;gBACd,QAAQ,CAAC,cAAc,CAAC,cAAc,CAAC,CAAC,SAAS;oBAC7C,IAAI,CAAC,KAAK,CAAC,GAAG,CAAC,IAAI,CAAC,CAAC,QAAQ,EAAE,CAAC;gBACpC,KAAK,CAAC;YACR;gBACE,KAAK,CAAC;QACV,CAAC;IACH,CAAC,CAAC"}
```

Comment:
These are generated files, right? I don't need to review them?

Reply:
Correct. You really only need to review any HTML, `*.ts` and `*.go` files.